build.sbt

```diff
@@ -1,6 +1,19 @@
-name := "spark-web-model"
+lazy val root = project.in(file("."))
+  .settings(Seq(
+    name := "spark-web-model",
+    mainClass := Some("com.snowplowanalytics.snowplow.webmodel.Main"),
+    organization := "com.snowplowanalytics",
+    version := "0.1.0-rc1",
+    scalaVersion := "2.11.12"
+  ))
+  .settings(BuildSettings.assemblySettings)
+  .settings(BuildSettings.buildSettings)
+  .settings(
+    libraryDependencies ++= Seq(
```

Comment:
This is most important thing that you should now about build definition. This is how you add library dependencies to your project (they're all listed in `project/Dependencies.scala`)